apps/arweave/src/ar_mining_server.erl

```diff
@@ -1079,6 +1084,13 @@ pick_hashing_thread(Threads) ->
 	{{value, Thread}, Threads2} = queue:out(Threads),
 	{Thread, queue:in(Thread, Threads2)}.
 
+optimal_performance(StoreID) ->
+	VdfSpeed = persistent_term:get(vdf_speed_ms) / 1000,
+	case prometheus_gauge:value(v2_index_data_size_by_packing, [StoreID, spora_2_6]) of
+		undefined -> 0;
+		StorageSize -> (100 / VdfSpeed) * (StorageSize / ?PARTITION_SIZE)
```

Comment:
We should replace 100 with 200 here because we've got 2 100 MiB recall ranges and thus in the best case every partition receives 200 MiB worth of chunks per second on average. Also, it is possible to set up a storage module smaller than a partition so makes sense to replace ?PARTITION_SIZE with the storage module size.

Reply:
Changed it look for the storage module size